core/cmake/BareosInstallConfigFiles.cmake

```diff
@@ -150,8 +150,7 @@ macro(BareosInstallConfigFiles CONFDIR CONFIGBASEDIRECTORY PLUGINS BACKENDS
     message(STATUS "install config files for PLUGIN ${PLUGIN}")
 
     # if plugin has -, conf.d directory is: python-ovirt ->
-    # python/ovirt/python-ovirt-conf.d else it is : cephfs ->
-    # cephfs/cephfs-conf.d
+    # python/ovirt/python-ovirt-conf.d else it is :
```

Comment:
Yeah I'm not sure that even our actual master code is in a comprehensive states, maybe since the clang-format one
I will have to check this one.

commit 0f01df3131
```diff
-    # if plugin has -, conf.d directory is:
-    # python-ovirt -> python/ovirt/python-ovirt-conf.d
-    # else it is :
-    # cephfs -> cephfs/cephfs-conf.d
+    # if plugin has -, conf.d directory is: python-ovirt ->
+    # python/ovirt/python-ovirt-conf.d else it is : cephfs ->
+    # cephfs/cephfs-conf.d
```

Reply:
I will goes for something like
```diff
-    # if plugin has -, conf.d directory is: python-ovirt ->
-    # python/ovirt/python-ovirt-conf.d else it is :
+/* clang-format off */
+    # if plugin has - in name, conf.d directory is:
+    # python-ovirt -> python/ovirt/python-ovirt-conf.d
+    # else it is: vmware -> vmmare/vmmare-conf.d
+/* clang-format on */
```